### PR TITLE
Blacklist command/inhibitor

### PR DESCRIPTION
**Pull request by UnseenFaith**
Just made a small inhibitor/command to deal with people who don't deserve to use commands/people I don't feel like dealing with on a moderation level so I just remove their access to commands completely lol. Might be useful for other people... could also extend this to per-server-confs, just a thought.

### CODE REVIEW

**Comment by CyberiumShadow** [comment]
I think now that per-server confs are here. We can forgo the blacklist.json and integrate them into the guild confs. We just gotta wait for confs to fully work.

**Comment by UnseenFaith** [comment]
Tis a good point, will work on this whenever I get per-server-confs working or when they work on install

**Comment by UnseenFaith** [comment]
Updated PR with per-server-conf versions of both the inhibitor and command.

File: commands/Moderation/blacklist.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/** Run Blacklists Inhibitor
+  * @param {Object} client - The Discord.js client
+  * @param {Object} msg - The command message
+  * @param {Object} cmd - The command
+  * @returns Reject() or Resolve(), depending on if the user is blacklisted or not.
+  */
+
+exports.run = (client, msg, cmd) => {
+    return new Promise((resolve, reject) => {
+      fs.readJson(`bwd/conf/${msg.guild.id}.json`, (err, guildConf) => {
```
Comment (hkwu) [inline commands/Moderation/blacklist.js]: You shouldn't need to use `fs` to read the confs. `client.funcs.confs` has a getter function to return a guild conf.

File: inhibitors/blacklist.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/** Run Blacklists Inhibitor
+  * @param {Object} client - The Discord.js client
+  * @param {Object} msg - The command message
+  * @param {Object} cmd - The command
+  * @returns Reject() or Resolve(), depending on if the user is blacklisted or not.
+  */
+
+exports.run = (client, msg, cmd) => {
+    return new Promise((resolve, reject) => {
+      fs.readJson(`bwd/conf/${msg.guild.id}.json`, (err, guildConf) => {
```
Comment (hkwu) [inline inhibitors/blacklist.js]: Same as `commands/Moderation/blacklist.js`.

**Comment by eslachance** [comment]
Thanks for the review @hkwu . Definitely correct, I have functions for that :D

**Comment by UnseenFaith** [comment]
Yeah, I realized this last night. I'm going to update it to properly use the functions Evie made.. For whatever reason I didn't think that the functions would actually write to the file, hence why I did read/white.. nevertheless I'm going to fix it and commit it tonight.

**Comment by CyberiumShadow** [comment]
@UnseenFaith Just letting you know that `<Message>.guildConf` can be used instead of needing to use `<Client>.funcs.confs.get(msg.guild)`

**Comment by UnseenFaith** [comment]
Fixed the spaghetti code and updated both to use the latest Komada version (v.0.8.2 at the time of this writing). Hopefully the last commit that I have to do, but please do point out any more issues so I can fix them.

**Comment by CyberiumShadow** [comment]
@UnseenFaith Any updates to this now that your Komada PR has been merged?

**Pull request closed without merging**